modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-order-impl/src/main/java/com/liferay/headless/commerce/delivery/order/internal/resource/v1_0/PlacedOrderResourceImpl.java

```diff
@@ -112,7 +112,7 @@ public Page<PlacedOrder> getChannelAccountPlacedOrdersPage(
 
 		AccountEntry accountEntry =
 			_accountEntryService.fetchAccountEntryByExternalReferenceCode(
-				contextCompany.getCompanyId(), accountExternalReferenceCode);
+				accountExternalReferenceCode, contextCompany.getCompanyId());
```

Comment:
same here, we can replace this with the get but leaving it in this "place"

Reply:
because we are using already the local service I think we can replace also the fetchCommerceChannel with the get and in this way we should be able to have the get of the account as first